flightdata-kafka-consumer/src/main/java/com/jackdaw/Runner.java

```diff
@@ -2,22 +2,35 @@
 
 import com.jackdaw.avro.flights.Flight;
 import com.jackdaw.consumer.FlightDataKafkaConsumer;
+import com.jackdaw.consumer.GeoJSONKafkaProducer;
+import com.jackdaw.consumer.JSONSerializer;
+import io.confluent.kafka.serializers.KafkaAvroDeserializerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
 
 import java.io.FileInputStream;
 import java.io.IOException;
+import java.util.Collections;
 import java.util.Properties;
 
 public class Runner {
     public static void main(String[] args) throws IOException {
-
         Properties topicProperties = new Properties();
         Properties schemaProps = new Properties();
 
         topicProperties.load(new FileInputStream("/volume/topic.properties"));
         schemaProps.load(new FileInputStream("/volume/flightdata-kafka-consumer.properties"));
+        schemaProps.put(KafkaAvroDeserializerConfig.SPECIFIC_AVRO_READER_CONFIG, true);
```

Comment:
Might be a good idea to be consistent, therefore move this to flightdata-kafka-consumer.properties.